docs/application/dotnet/guides/app-management/cion.md

```diff
@@ -0,0 +1,434 @@
+        {
+            DiscoveredEventArgs eventArgs = new DiscoveredEventArgs(peerInfo);
+            DiscoveredEvent?.Invoke(this, eventArgs);
+        }
+
+        protected override void OnPayloadReceived(Payload payload, PayloadTransferStatus status)
+        {
+            PayloadReceivedEventArgs eventArgs = new PayloadReceivedEventArgs(payload, status);
+            PayloadReceivedEvent?.Invoke(this, eventArgs);
+        }
+
+        public class ConnectionResultEventArgs : EventArgs
+        {
+            public ConnectionResultEventArgs(PeerInfo peerInfo, ConnectionResult result)
+            {
+                PeerInfo = peerInfo;
+                Result = result;
+            }
+            public PeerInfo PeerInfo { get; }
+            public ConnectionResult Result { get; }
+        }
+
+        public class DisconnectedEventArgs : EventArgs
+        {
+            public DisconnectedEventArgs(PeerInfo peerInfo)
+            {
+                PeerInfo = peerInfo;
+            }
+            public PeerInfo PeerInfo { get; }
+        }
+
+        public class DiscoveredEventArgs : EventArgs
+        {
+            public DiscoveredEventArgs(PeerInfo peerInfo)
+            {
+                PeerInfo = peerInfo;
+            }
+            public PeerInfo PeerInfo { get; }
+        }
+
+        public class PayloadReceivedEventArgs : EventArgs
+        {
+            public PayloadReceivedEventArgs(Payload payload, PayloadTransferStatus status)
+            {
+                Payload = payload;
+                Status = status;
+            }
+            public Payload Payload { get; }
+            public PayloadTransferStatus Status { get; }
+        }
+    }
+    ```
+    ```csharp
+    string serviceName = "TestService";
+    var client = new CionClient(serviceName);
+    ```
+    
+
+2. Discover the server to connect. If the server is discovered, connect to discovered server:
+
+    ```csharp
+    client.TryDiscovery();
+
+    client.DiscoveredEvent += (sender, e) =>
+    {
+        CionClient.DiscoveredEventArgs eventArgs = e as CionClient.DiscoveredEventArgs;
+        client.Connect(eventArgs.PeerInfo);
+    };
+    ```
+
+3. Send data and payload:
+
+    ```csharp
+    string TestMessage = "Test Message";
+
+    client.SendData(Encoding.UTF8.GetBytes(TestMessage), 5000);
+    ```
+    ```csharp
+    string TestMessage = "Test Message";
+    DataPayload dataPayload = new DataPayload(Encoding.UTF8.GetBytes(TestMessage));
+
+    PayloadAsyncResult result = await client.SendPayloadAsync(dataPayload);
+    ```
+
+Cion server can listen the request from the client. If a connection request comes from the client,the server can accept connection.
+After connection is accepted,the server can receive data and payload.
+
+To communicate with client, use the [Tizen.Applications.Cion.ServerBase](https://docs.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.Cion.ServerBase.html) class as follows:
```

Comment:
Same as above comment (the symbolic link "latest" need to be fixed)

Reply:
link is changed to API9 now

Reply:
changed to latest